### ISSUE TITLE
selected_char_kind cannot be imported from module

### ISSUE THREAD

**Issue by clementval** [omni-compiler/omni-compiler#181]
```fortran 
MODULE char_kind_module
  INTEGER, PARAMETER :: R1 = SELECTED_REAL_KIND(13)
  INTEGER, PARAMETER :: C1 = SELECTED_CHAR_KIND('ascii')
END MODULE char_kind_module
```

```fortran 
module test_iso
  use char_kind_module
end module test_iso
```

This will results in a segfault as `selected_char_kind`is not considered as an intrinsic function.

**Comment by clementval** [comment]
Should be fix by #182

**Comment by clementval** [comment]
Fix by #182
